hw8_dictionary.py

```diff
@@ -0,0 +1,46 @@
+# Задача 1: Є система, де є списки користувачів.
+# roles = { 'admin' : [...], 'maintainer' : [...], 'manager' : [...], 'developer': [...]}
+# - в цих списках треба понаписувати імена користувачів.
+# При введенні імені необхідно перевірити, до якої ролі належить ця людина та вивести повідомлення виду:\
+#     "Hello, <role>"
+# Якщо імені немає у списках, вивести: "Hello, Guest"
+
+roles = {'admin': [...], 'maintainer': [...], 'manager': [...], 'developer': [...]}
+for key, value in roles.items():
+    roles[key] = input(f'Enter name {key}:')
+request_name = input('Enter request name:')
+if not (request_name in roles.values()):
+    print(f'Hello, Guest')
+else:
+    for key, value in roles.items():
+        if value == request_name:
+            print(f'Hello, {key} ')
+            break
```

Comment:
It should be simpler for your implementation
![image](https://github.com/EvgeniyRud/Hillel_project/assets/51053483/900145af-5868-43bd-ac3d-471b778e72b2)